### ISSUE TITLE
Dossier

### ISSUE THREAD

**Issue by lakonis** [timoguic/sp_hub#29]
**Un dossier est défini par :**

- un titre de dossier
- un ou plusieurs rédacteurs `<div vocab="http://xmlns.com/foaf/0.1/" typeof="Person" class="foaf-redactor">`
- un article de type “sommaire” `<meta name="DC.type" class="typeArticle" content="Sommaire dossier" />`
- une liste ordonnée d’articles.

__Règles :__

- Le titre/ss-titre et le.s auteur.s de l’article sommaire ~sont respectivement~ ne correspondent pas nécessairement le titre/ss-titre du dossier et les rédacteurs du dossier.
- Dans un article, la balise suivante  `<span property="isPartOf" typeof="PublicationVolume" resource="#periodical" class="titreDossier">Corps contemporain et espace v&#233;cu</span>` nous indique deux choses : 
  1. si l'article appartient à un dossier
  2. le titre du dossier.
- Un article peut appartenir à plusieurs dossiers.

**Fonctionnalités sur sphub :**
- édition de tous les éléments du dossier : 
  - titre, rédacteur.s, ajout/suppression d'article, ordre des articles
- création d'un dossier "from scratch" (sans données html préalable)

**Comment by lakonis** [comment]
Si un article appartient à un dossier, on a la donnée suivante :
```
<!--Indexation dossier-->
<span property="isPartOf" typeof="PublicationVolume" resource="#periodical" class="titreDossier">Ontologie du num&#233;rique</span>
```

**Comment by lakonis** [comment]
@ServanneMo  : confirmer le fait que le titre du dossier peut être différent du titre de l'article sommaire, comme pour Ontologie du numérique.. 
Pour le dossier SP1271, les titres du _dossier_ et  du_sommaire_ sont bien les mêmes.

**Comment by lakonis** [comment]
@ServanneMo également deux types d'article dans le yaml pour le sommaire de Ontologie du numérique. or le type "sommaire de dossier" est passé à la trappe dans les données html. Important de ne conserver qu'un seul élément.

**Comment by ServanneMo** [comment]
Oui, on gardera seulement "sommaire du dossier". Les éditeurs doivent se méfier. J'ajoute dans le protocole. Sinon oui bien sûr, les titres peuvent être différents, surtout si on a "intro" ou une autre mention du même genre. Ce n'est pas un problème.

**Comment by lakonis** [comment]
Merci @ServanneMo je met à jour les specs ci-dessus.

**Comment by timoguic** [comment]
@lakonis plus d'infos sur ce qui représente le "sous-titre" dans le HTML?

**Comment by timoguic** [comment]
Dans le corpus de documents HTML, je note la présence (suspecte?) de deux dossiers:
 * Corps contemporain et espace vécu (SP1257, SP1263, ...)
 * Corps contemporain et espace vécu : entre imaginaire et expérience (SP1259)

**Comment by lakonis** [comment]
Il y a une balise subtitle dans les méta. Nous allons l'étendre aux autres
balises DC.
Pour le dossier c'est une erreur, un éditeur a compilé titre et sous-titre
du dossier dans le title.
On va corriger ça.

**Comment by lakonis** [comment]
les balises subtitle ont été ajoutées au template. Nous devons réexporter
les html pour les prendre en compte.

# Nicolas Sauret
* +1 438 404 5993 - @nicolasauret <https://twitter.com/nicolasauret> -
nicolassauret.net
* doctorant Paris Ouest (laboratoire Dicen-IDF <http://www.dicen-idf.org>),
Université de Montréal (CRC Écritures numériques
<http://ecrituresnumeriques.ca/>)
* co-organisateur du séminaire Écritures numériques et éditorialisation
<http://seminaire.sens-public.org>
